/**:
  ros__parameters:
    rate: 200.
    x0: -0.5 
    y0: 0.7 
    theta0: 1.28 
    obstacles/x: [-0.5, 0.8, 0.4]
    obstacles/y: [-0.7, -0.8, 0.8]
    obstacles/r: 0.038
    arena_x_length: 5.0
    arena_y_length: 5.0